core/iwasm/fast-jit/jit_frontend.c

```diff
@@ -683,7 +693,7 @@ jit_compile_func(JitCompContext *cc)
                 for (i = 0; i <= br_count; i++)
                     read_leb_uint32(frame_ip, frame_ip_end, br_depths[i]);
 
-                if (!jit_compile_op_br_table(cc, br_depths, br_count,
+                if (!jit_compile_op_br_table(cc, br_depths, br_count + 1,
```

Comment:
The original code is ported from core/iwasm/compilation/aot_compiler.c:
https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/iwasm/compilation/aot_compiler.c#L282-L283
Could we not change it and make it similar to aot_compiler.c? We can increase br_count in jit_emit_control.c

Reply:
Sure we can keep following AOT code. 

Just a little wired if the length parameter (br_count) is actually longer than the size of the array(br_depths).